src/components/fx/attributes.js

```diff
@@ -33,6 +33,13 @@ module.exports = {
             family: extendFlat({}, fontAttrs.family, { arrayOk: true }),
             size: extendFlat({}, fontAttrs.size, { arrayOk: true }),
             color: extendFlat({}, fontAttrs.color, { arrayOk: true })
+        },
+        namelength: {
+            valType: 'number',
+            min: 0,
+            dflt: 15,
+            role: 'style',
```

Comment:
We should try adding `arrayOk` support here, so that 

```js
trace = {
  x: [1, 2, 3],
  y: [2, 1, 2],
  namelength: [15, 10, 40]
}
```
would work.